cluster-up/cluster/k3d-1.25-sriov/config_sriov_cluster.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
Nothing really here is specific to  `1.25` can we move it to a common `k3d` directory ? if we see is not the case with future k3d versions we can duplicate.

Reply:
I  prefer to keep in this PR the file structure the same it was with kind, later on we can refactor
it answers also 
`First pass, in general can we move the sriov things to cluster-up/cluster/k3d/sriov.sh ?`